FHIR/GoFSH
# Avoid error logging for non-FHIR XML

# Description
Fixes https://github.com/FHIR/GoFSH/issues/119.

When the package we used to parse XML FHIR files encounters a non-FHIR XML file, it would log an error like:
```
error Could not load FHIR-us-core.xml: Unknown resource type: specification
```
But when we encounter non-FHIR JSON, we don't log an error, but instead just assume we are supposed to skip this file, and log a debug message. This PR updates the XML handling so that when we encounter non-FHIR XML, we now just log a debug. We can't do this directly, since the package is kind of a black box for us, so instead we just intercept the message coming from the package and log a debug.

# Testing
To test this, you can do the following:
* On `master`, run GoFSH on a folder which contains XML files that are not FHIR. Make sure to process the XML by using `-t json-and-xml`. You should see errors like the one shown above.
* On this branch, run GoFSH on the same folder. The errors should go away (but any generated FSH should not change)
* On this branch, run GoFSH on the same folder, but with `-l debug`, and note the debug messages referring to the invalid XML.
